easybuild/easyconfigs/n/NEMO/NEMO-3.4-r5541-ictce-6.1.5.eb

```diff
@@ -0,0 +1,22 @@
+name = 'NEMO'
+version = '3.4-r5541'
+
+homepage = 'http://www.nemo-ocean.eu/'
+description = """NEMO (Nucleus for European Modelling of the Ocean) is a state-of-the-art modeling framework for
+ oceanographic research, operational oceanography seasonal forecast and climate studies."""
+
+toolchain = {'name': 'ictce','version': '6.1.5'}
+toolchainopts = {'opt': False, 'optarch': False}
+
+# authenticated SVN access required
+# svn --username 'login' co http://forge.ipsl.jussieu.fr/nemo/svn/branches/2012/dev_v3_4_STABLE_2012
+# tar cfvz NEMO-3.4-r<rev>.tar.gz dev_v3_4_STABLE_2012
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('netCDF-Fortran', '4.2')]
+
+with_components = ['OPA_SRC', 'LIM_SRC_3']
+add_keys = ['key_iomput', 'key_mpp_mpi', 'key_dtasal', 'key_dtatem', 'key_tradmp', 'key_lim3', 'key_orca_r1=75']
+del_keys = ['key_lim2', 'key_zdftmx', 'key_orca_r2']
```

Comment:
@omula: how were these picked? can we come up with a reasonable `versionsuffix` that describes this particular build?